docs/pages/access-controls/reference.mdx

```diff
@@ -334,10 +334,19 @@ who created the session.
 
 <Details title="Version Warning: Before 8.1" opened={false}>
 Teleport versions prior to 8.1 don't support the roles exemplified below.
-You may to create such roles after an upgrade, but in the event of a cluster
+You may create such roles after upgrading, but in the event of a cluster
 downgrade they will become invalid.
 </Details>
 
+<Admonition
```

Comment:
At first, I thought this Admonition was about the preset `auditor` role, but the emphasis is actually on the second sentence. Can we combine the two sentences to streamline this (or rearrange the sentences)? Maybe:

```markdown
In order for the following roles to take effect you will need to ensure your user doesn't also have a more permissive role, such as the preset `auditor` role, that allows access to all events, sessions, and session recordings.
```

Reply:
Yeah, the second sentence is the important part, but the reason people get caught off guard is because almost everyone applies the auditor role by default.